theme/static/js/custom.js

```diff
@@ -175,12 +174,12 @@ $(document).ready(function () {
         createResource(resourceType, title);
     });
     
-    function createResource(type, title="Untitled Resource") {
+    function createResource(type, title) {
         // Disable dropdown items while we process the request
         $(".navbar-inverse .res-dropdown .dropdown-menu").toggleClass("disabled", true);
 
         var formData = new FormData();
-
+        title = "Untitled Resource";
```

Comment:
I think putting "Untitled Resource" as the default for title argument is better since it allows createResource() to be called without passing a value to title. Making this change would not allow calling createResource() without passing in a value to title any more.

Reply:
That crashes Internet Explorer

Reply:
Sidenote for any non-Windows developers - https://www.lambdatest.com/test-on-internet-explorer-11-browser-online has been useful for me to test IE